src/Renderer/LayeredMaterial.js

```diff
@@ -172,14 +171,6 @@ LayeredMaterial.prototype.dispose = function dispose() {
             }
         }
     }
-
-    var jT = new JavaTools();
-
-    jT.freeArray(this.textures[0]);
```

Comment:
Anyway, `dispose` is never called, and *generally*, it's better to let the gc dispose of everything (if there is no event listeners). It will optimize natively, pretty sure it is always better than doing it explicitly. And if not, it's a browser bug :-)